templates/web/base/report/display_tools_mobile.html

```diff
@@ -0,0 +1,43 @@
+<div class="shadow-wrap">
```

Comment:
Sorry, one more thing - this file has some IDs in that are duplicating the ones in display_tools.html. I see you fixed the ones that you were then using in the JS, but the others will need fixing too - e.g. key-tool-report-abuse (not used at all, can be removed), key-tool-problems-nearby (does have some uses), and remove-from-site-form (not needed at all?)